protected/humhub/modules/admin/controllers/PendingRegistrationsController.php

```diff
@@ -125,6 +125,51 @@ public function actionDelete($id)
         return $this->render('delete', ['model' => $invite]);
     }
 
+    /**
+     * Delete all invitations
+     *
+     * @param integer $id
+     * @return string
+     * @throws HttpException
+     * @throws \Throwable
+     */
+    public function actionDeleteAll()
+    {
+        if (Yii::$app->request->isPost) {
+               Invite::deleteAll();
+            $this->view->success(Yii::t(
+                'AdminModule.user',
+                'Deleted all invitations'
+            ));
+        }
+        return $this->redirect(['index']);
+    }
+    /**
```

Comment:
\n

Reply:
@luke- I am sorry did not catch what does it mean...

Reply:
@MasterHans There is a missing newline between the end of the method and the next one.

Reply:
Done!